doc/sphinx/user/extending/plugin-types/boundary-heat-flux.md

```diff
@@ -0,0 +1,34 @@
+
+# Boundary heat flux
+
+In contrast to prescribing the actual temperature at a boundary, it is
+also possible to prescribe the *heat flux* across a boundary. A number
+of heat flux models are already implemented, but if you want to
+implement a new boundary heat flux model, you need to overload the
```

Comment:
I never heard the term "to overload a class" before. Do you mean 'implement (or overload) the functions in the interface class"?